react/modules/imageAsList.tsx

```diff
@@ -6,7 +6,8 @@ import type { ImagesSchema } from '../ImageTypes'
 export const getImagesAsJSXList = (
   images: ImagesSchema,
   isMobile: boolean,
-  height: string | number
+  height: string | number,
+  preload?: boolean
```

Comment:
What do you think about switch this property to an object with it? This may help in maintainability and readability. Also, it's ok to still as it is.